### ISSUE TITLE
Feature: Assert.IsNull / Assert.IsNotNull Classic to Fluent

### ISSUE THREAD

**Issue by jcurl** [nunit/nunit.analyzers#163]
Thanks for the good analyzer. In my old project, I'm migrating to new style before upgrading to NUnit 3.x (no fixed date yet, but when I'm ready). 

A see no warnings for Assert.IsNull, Assert.IsNotNull.

## Expected
`Assert.IsNull(foo)` => `Assert.That(foo, Is.Null)`
`Assert.IsNotNull(foo)` => `Assert.That(foo.Is.Not.Null)`
`Assert.Null(foo)` => `Assert.That(foo, Is.Null)`
`Assert.NotNull(foo)` => `Assert.That(foo.Is.Not.Null)`

**Comment by mikkelbu** [comment]
Sounds like an analyzer to me :).

**Comment by jcurl** [comment]
I've had a shot at this, but I'm not familiar enough with Roslyn to do the code fix for Assert.That(obj, Is.Not.Null). 

Please take a look at 35819be800b8388fd720e48c22a5cd33d29d8ffe

**Comment by Dreamescaper** [comment]
@jcurl 
I think it should be similar as existing `AreNotEqualClassicModelAssertUsageCodeFix`, 
but probably with something similar as that:
```csharp
            arguments.Insert(2, SyntaxFactory.Argument(
                    SyntaxFactory.MemberAccessExpression(
                        SyntaxKind.SimpleMemberAccessExpression,
                        SyntaxFactory.MemberAccessExpression(
                            SyntaxKind.SimpleMemberAccessExpression,
                            SyntaxFactory.IdentifierName(NunitFrameworkConstants.NameOfIs),
                            SyntaxFactory.IdentifierName(NunitFrameworkConstants.NameOfIsNot)),
                        SyntaxFactory.IdentifierName(NunitFrameworkConstants.NameOfIsNull))));
```

(haven't checked, but should work)

or even that:
```csharp
            arguments.Insert(2, SyntaxFactory.Argument(
                SyntaxFactory.ParseExpression($"{NameOfIs}.{NameOfIsNot}.{NameOfIsNull}")));
```

(I often use SharpLab to investigate Syntax Tree, e.g. see [here](https://sharplab.io/#v2:C4LghgzsA0AmIGoA+ABATARgLACgUGYACdQgYUIG9dCbiiUAWQgWQAoBKQ62qnW/wgEEIEAKYAnYADoAKgAswwVmADGwAK5gANtEIBJCFIByAe2lH1WrewDc3GgF9cDoA===))

**Comment by mikkelbu** [comment]
Closed by #168
